src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -508,22 +387,29 @@ public function READ_args() {
 	}
 
 	/**
-	 * Parses the arguments populated parsing the request filling out with the defaults.
-	 *
 	 * @param array $args
-	 * @param array $defaults
 	 *
-	 * @return array
+	 * @return int
 	 */
-	protected function parse_args( array $args, array $defaults ) {
-		foreach ( $this->supported_query_vars as $request_key => $query_var ) {
-			if ( isset( $defaults[ $request_key ] ) ) {
-				$defaults[ $query_var ] = $defaults[ $request_key ];
-			}
-		}
+	protected function get_total( $args ) {
+		$this->total = count( tribe_get_events( array_merge( $args, array(
+			'posts_per_page'         => - 1,
```

Comment:
Not necessary to change, but the space between the `-` and the `1` is kind of weird.